corehq/apps/style/templates/style/base.html

```diff
@@ -276,7 +276,7 @@
                                     </div>
                                 {% endfor %}
                             {% endif %}
-                            <div id="message-alerts" data-bind="foreach: alerts">
+                            <div id="message-alerts" class="ko-template" data-bind="foreach: alerts">
```

Comment:
I think that makes sense; `ko-template` is more semantic than `hide`. Although, `ko-template` could also mean semantic things other than "don't want people to be able to see this util js has loaded and its behavior is available," and there may be other widgets that don't use knockout but that do still fall into the category of "useless/distracting until js has loaded". But that's getting hair-splitting hairs over semantics, and for the sake of this PR `ko-template` seems fine.